doc/src/installation.rst

```diff
@@ -0,0 +1,161 @@
+.. highlightlang:: none
+
+************
+Installation
+************
+
+Quick-start
+===========
+
+PyFR |release| can be installed using `pip <https://pypi.python.org/pypi/pip>`_
+and `virtualenv <https://pypi.python.org/pypi/virtualenv>`_, as shown in the
+quick-start guides below.
+
+Alternatively, PyFR |release| can be installed from 
+`source <http://www.pyfr.org/download.php>`_, see :ref:`compile-from-source`.
```

Comment:
We probably want to update this URL.

Reply:
Good catch. We should point to the Github repo now.